olm-catalog/serverless-operator/project.yaml

```diff
@@ -4,18 +4,19 @@ project:
     # all components in `dependencies.previous` to the same versions as `dependencies` in the same PR.
     # Otherwise, the upgrade tests will not pass, as we have a different SO version with the same bundle contents.
     # Also make sure to update values under `olm.previous` by copying from `olm.replaces` and `olm.skipRange`.
-    version: 1.29.0
+    version: 1.30.0
 olm:
-    replaces: 1.28.0
-    skipRange: '>=1.28.0 <1.29.0'
+    replaces: 1.29.0
+    skipRange: '>=1.29.0 <1.30.0'
     channels:
         default: 'stable'
         list:
-            - 'stable'
-            - 'stable-1.29'
+            - stable
+            - stable-1.29
+            - stable-1.30
```

Comment:
will this grow endlessly?

Reply:
This should stop growing once we have the 3 supported versions (due to the new channel naming strategy)